app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/triage/MultipleDisposalsTriageController.scala

```diff
@@ -1032,6 +1088,14 @@ class MultipleDisposalsTriageController @Inject() (
       }
     }
 
+  // TODO: what is the possibility to capture other TaxYearExchanged values like TaxYearBefore2020, DifferentTaxYears
```

Comment:
I suppose this is wip?

Reply:
Maybe let's code it some previous years for the before 2020 and different tax years to map to the radios we need?

Reply:
mmm I have noted that TODO in the beginning. But now it's working as expected. 
Removed that TODO